src/app/dataset-block/metadata-block/components/event-details/event-details.component.html

```diff
@@ -39,6 +39,14 @@
         <app-set-watermark-event [event]="setWatermarkEvent"></app-set-watermark-event>
     </ng-container>
 
+    <ng-container *ngIf="isAddPushSourceEvent">
```

Comment:
Can you remind why we are not using dynamic components to implement variation here? I remember some issues with clipboard copying.

Reply:
We solved the problem with clipboard copying. We installed a third party library for this